docs/overview.adoc

```diff
@@ -953,13 +953,58 @@ Sometimes type information gets lost, these functions help you to coerce an "Any
 | apoc.node.degree.in(node, relationshipName) | returns total number of incoming relationship
 | apoc.node.degree.out(node, relationshipName) | returns total number of outgoing relationship
 | apoc.node.labels(node) | returns labels for (virtual) nodes
+| apoc.nodes.collapse([<node>], {config}) | returns the list of nodes merged into the first
 | apoc.rel.id(rel) | returns id for (virtual) relationships
 | apoc.rel.type(rel) | returns type for (virtual) relationships
 | apoc.any.properties(node/rel/map, [[keys]]) | returns properties for virtual and real, nodes, rels and maps, optionally restrict via keys
 | apoc.any.property(node/rel/map) | returns property for virtual and real, nodes, rels and maps
 | apoc.create.uuid() | returns a UUID string
 |===
 
+On `apoc.nodes.collapse` with config properties you can choose from 3 different behavior:
+ * "properties": "overwrite" : if there is the same property in more node, in the new one will have the last relationship's/node's property value
+ * "properties": "discard" : if there is the same property in more node, the new one will have the first relationship's/node's property value
+ * "properties": "combine" : if there is the same property in more node, the new one a value's array with all relationship's/node's values
+
+Relationships properties are managed with the same nodes' method, if properties parameter isn't set relationships properties are combined.
+
+On `apoc.nodes.collapse` there are many config param:
+ * "mergeRels: true/false" : give the possibility to merge relationships with same type and direction.
+ * "selfRel: true/false" : give the possibility to create the self relationship.
+ * "countProperties: true/false" : give the possibility count all the properties merged for nodes and relationships.
+
+e.g:
+
+With this dataset we have:
+
+image::{img}/apoc.nodes.collapse_1.png[width=600]
+
+call the apoc as:
+----
+MATCH (p:Person)-[:LIVES_IN]->(c:City)
+WITH c, collect(p) as subgraph
+CALL apoc.nodes.collapse(subgraph,{properties:'combine', mergeRels:true, countProperties: true}) yield from, rel, to
+return from, rel, to
+----
+we have this result:
+
+image::{img}/apoc.nodes.collapse_2.png[width=600]
+
+With this dataset we have:
+
+image::{img}/apoc.nodes.collapse_3.png[width=600]
+
+call the apoc as:
+----
+MATCH (p:Person)-[:LIVES_IN]->(c:City)
+WITH c, c + collect(p) as subgraph
+CALL apoc.nodes.collapse(subgraph,{properties:'ignore', mergeRels:true, countProperties: true}) yield from, rel, to
```

Comment:
"ignore" is not a documented option, see above. there is only discard or overwrite.